src/Microsoft.DotNet.ScenarioTests.Common/Program.cs

```diff
@@ -7,10 +7,9 @@
 using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
-using System.ComponentModel;
-using System.Diagnostics;
 using System.Runtime.InteropServices;
 using System.Collections.Immutable;
+using Microsoft.DotNet.ScenarioTests.SdkTemplateTests;
```

Comment:
This shouldn't be necessary if you move the `OperatingSystemFinder` in here.

Reply:
Removed